src/Kliva/Models/ActivitySummaryService.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using DynamicData;
+using DynamicData.Binding;
+using GalaSoft.MvvmLight.Threading;
+using Kliva.Services.Interfaces;
+
+namespace Kliva.Models
+{
+    public class ActivitySummaryService : IDisposable
+    {        
```

Comment:
Yeah, I kinda chucked all the classes in one place, so it's probably better somewhere else. I would still be inclined to call it a service though (or cache), add not a factory: now you have the ability to bind to the feed in different places if there's ever a need, sharing the same underlying cache.